### ISSUE TITLE
question

### ISSUE THREAD

**Issue by simonchen007** [j0lama/Wecast#1]
Hello,

Does those roms are dump from the Wecast ?
It seems an Android system , and without GPU library .

**Comment by j0lama** [comment]
Yes, those roms were extracted from the Wecast. I think Wecast run a reduce Linux version.

**Comment by simonchen007** [comment]
so the Android can run without GPU library  ? The ROMs seems small very much.

**Comment by j0lama** [comment]
I don't think Wecast runs Android.
I'm not sure either if Wecast OS does not contain any GPU libs. It seems weird to me that all the graphics are rendered by the CPU.

**Comment by simonchen007** [comment]
I got the SDK of rk3036 just now, and comfirm it's running Android5.1 , 
Does your wecast works for IOS ?
